blockchains/smartchain/assets/0x0567D7fad179dCd2a89508B77cCe93E26f8998F7/info.json

```diff
@@ -0,0 +1,6 @@
+{
+    "name": "VIDTC Luxury goods",
+    "website": "https://about.v-id.org",
+    "short_description": "NFTs for watches, fashion & jewelry",
+    "explorer": "https://bscscan.com/address/0x0567d7fad179dcd2a89508b77cce93e26f8998f7"
```

Comment:
Please update it to `https://bscscan.com/token/0x0567d7fad179dcd2a89508b77cce93e26f8998f7`